scripts/update/0.5-0.6/update.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+kubectl create ns update-host-files --dry-run=client -o yaml | kubectl apply -f -
+kubectl create -n update-host-files cm script --from-file "${script_dir}"/cluster.sh --dry-run=client -o yaml | kubectl apply -f -
+
+for nodename in $(kubectl get nodes -ojsonpath='{.items[*].metadata.name}')
+do
+  node_labels=$(kubectl get node $nodename --show-labels)
+  if echo "${node_labels}" | grep 'metallb.lokomotive.io/my-asn' > /dev/null; then
+    kubectl label node --overwrite $nodename lokomotive.alpha.kinvolk.io/bgp-enabled=true
+    mode="metallb"
+  elif echo "${node_labels}" | grep 'node.kubernetes.io/master' > /dev/null;  then
+    mode="controller"
+  else
+    mode="general"
+  fi
+  echo "
+apiVersion: v1
+kind: Pod
```

Comment:
- Chose pod over job because I wanted to do a scheduling on my own.
- Chose pod over daemonset because this is not something I want to run for eternity. 

I basically wanted scheduling of daemonset and run time of job, so chose a pod. Scheduling happens using a for loop and if a pod fails user can always see what happened, via logs.